packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.test.tsx

```diff
@@ -119,16 +132,63 @@ describe('CompleteResetPassword page', () => {
   });
 
   // TODO : check for metrics event when link is expired or damaged
-  it('emits the expected metrics on render when the link is valid', () => {
-    render(
-      <CompleteResetPassword
-        email={MOCK_ACCOUNT.primaryEmail.email}
-        linkStatus="valid"
-      />
-    );
-    expect(usePageViewEvent).toHaveBeenCalledWith(
+  it('emits the expected metrics on render', () => {
+    renderWithAccount({} as unknown as Account);
+
+    expect(logPageViewEvent).toHaveBeenCalledWith(
       'complete-reset-password',
       REACT_ENTRYPOINT
     );
   });
+
+  it('displays any errors', async () => {
+    account = {
+      resetPasswordStatus: jest.fn().mockResolvedValue(true),
+      completeResetPassword: jest
+        .fn()
+        .mockRejectedValue(new Error('Request failed')),
+    } as unknown as Account;
+
+    renderWithAccount(account);
+
+    await act(async () => {
+      fireEvent.input(screen.getByTestId('new-password-input-field'), {
+        target: { value: PASSWORD },
+      });
+    });
+
+    await act(async () => {
+      fireEvent.input(screen.getByTestId('verify-password-input-field'), {
+        target: { value: PASSWORD },
+      });
+    });
```

Comment:
I like to reference Jody's better testing PDF sometimes (I can send it to you if you need a link), around slide ~45 there's some notes about `act` and how `fireEvent` is already wrapped in an `act`:

![image](https://user-images.githubusercontent.com/13018240/219490600-f8923a04-18a0-4c8f-b39c-89a0d7632a35.png)

I think I've seen a case or two where it's needed but maybe some of these can be simplified?